seatunnel-e2e/seatunnel-e2e-common/src/test/java/org/apache/seatunnel/e2e/common/container/TestContainerId.java

```diff
@@ -27,7 +27,7 @@
 @Getter
 public enum TestContainerId {
     FLINK_1_13(FLINK, "1.13.6"),
-    SPARK_2_4(SPARK, "2.4.3"),
+    SPARK_2_4(SPARK, "2.4.6"),
```

Comment:
What's wrong with `2.4.3`?

Reply:
Because image is changed.

Reply:
the spark docker's image version number is already 2.4.6